dragonchain/lib/matchmaking.py

```diff
@@ -329,14 +329,18 @@ def make_matchmaking_request(
             return make_matchmaking_request(http_verb=http_verb, path=path, json_content=json_content, retry=False, authenticated=authenticated)
         elif retry and response.status_code == 403 and authenticated:
             _log.warning(
-                "[MATCHMAKING] received 403 from matchmaking. Registration is expired or dragon net config is invalid. Re-registering and trying again"
+                "[MATCHMAKING] received 403 from matchmaking. "
+                "Registration is expired or dragon net config is invalid. "
+                "Re-registering and trying again"
             )
             register(retry=False)
             return make_matchmaking_request(http_verb=http_verb, path=path, json_content=json_content, retry=False, authenticated=authenticated)
         elif response.status_code == 402:
             raise exceptions.InsufficientFunds("received insufficient funds (402) from matchmaking")
         elif response.status_code == 404:
             raise exceptions.NotFound("Not found (404) from matchmaking")
+        elif response.status_code >= 500:
+            raise exceptions.MatchmakingRetryableError(f"[MATCHMAKING] Server error {response.status_code} from matchmaking")
```

Comment:
Anywhere where anything was catching an `exceptions.MatchmakingError` probably needs to be able to catch this new new code too

Reply:
yeah, there was one spot in... L1 actions I think, which was catching a `MatchmakingError`, I added `MatchmakingRetryableError` to it

Reply:
Never mind, you got it